x-pack/plugins/session_view/public/components/SessionView/index.tsx

```diff
@@ -106,14 +108,18 @@ export const SessionView = ({ sessionEntityId, height }: SessionViewDeps) => {
     );
   };
 
-  if (!data.length) {
-    return renderNoData();
-  }
-
-  return (
-    <>
-      <EuiSearchBar query={searchQuery} onChange={onSearch} />
-      {data && (
+  const renderProcessTree = () => {
```

Comment:
i feel like we should move this logic into the `ProcessTree` component

Reply:
It depends, I would say

hopefully we get some designs for a loading state (i.e. using those shimmering boxes) 

personally I think it's fine to leave it out of the process component for now just to keep things simple. If we need to change the loading UI we can change it more easily